src/Microsoft.OpenApi.Readers/V3/OpenApiHeaderDeserializer.cs

```diff
@@ -45,12 +51,30 @@ internal static partial class OpenApiV3Deserializer
                     o.Style = n.GetScalarValue().GetEnumFromDisplayName<ParameterStyle>();
                 }
             },
+            {
+                "explode", (o, n) =>
+                {
+                    o.Explode = bool.Parse(n.GetScalarValue());
+                }
+            },
             {
                 "schema", (o, n) =>
                 {
                     o.Schema = LoadSchema(n);
                 }
-            }
+            },
```

Comment:
This is still missing the property "content". 
```suggestion
                },
                {
                    "content", (o, n) =>
                    {
                        o.Content = n.CreateMap(LoadMediaType);
                    }
                },
```

Reply:
Spec does say "content" should be allowed, but "content" is a map keyed of media-type, not sure if concept of media-types is applicable to headers. I feel spec is loose here.

Reply:
That's fair. I'm not certain how it will be used either, but I was just reading off the spec. We can leave this be.